app/services/imports/scheme_location_import_service.rb

```diff
@@ -0,0 +1,166 @@
+module Imports
+  class SchemeLocationImportService < ImportService
+    def create_scheme_locations(folder)
+      import_from(folder, :create_scheme_location)
+    end
+
+    def create_scheme_location(xml_document)
+      management_group = location_field_value(xml_document, "mgmtgroup")
+      schemes = Scheme.where(old_id: management_group)
+      raise "Scheme not found with legacy ID #{management_group}" if schemes.empty?
+
+      if schemes.size == 1 && schemes.first&.locations&.empty?
+        scheme = update_scheme(schemes.first, xml_document)
+      else
+        scheme = find_scheme_to_merge(schemes, xml_document)
+        scheme ||= duplicate_scheme(schemes, xml_document)
+      end
+      add_location(scheme, xml_document)
+    end
+
+  private
+
+    REGISTERED_UNDER_CARE_ACT = {
```

Comment:
Worth referencing the constant from the model here so that the values can't get out of sync?

Reply:
They are not the same values.
This is the label we use as a suffix on the scheme name.